microcosm_pubsub/tests/conventions/test_messages.py

```diff
@@ -23,13 +25,72 @@
     deleted,
     make_media_type,
 )
+from microcosm_pubsub.decorators import schema
 from microcosm_pubsub.tests.fixtures import ExampleDaemon, noop_handler
 
 
 class Foo:
     pass
 
 
+class TestEnum(Enum):
+    key = auto()
+
+    def __str__(self):
+        return self.name
+
+
+class EnumField(Field):
+    """
+    Test serialization of non-serializable fields
+    This is mostly taken from microcosm-flask's EnumField
```

Comment:
Either types are added to their own lib, or we depend on mc-flask. I opted for copying.